todoapp/controllers/user.controller.js

```diff
@@ -24,7 +24,7 @@ async function createUser(req, res) {
       });
     }
     body.password = await hashObject(body.password);
-    const newUser = new User({ ...body });
+    const newUser = new User({ ...body, photo: req.file.path });
```

Comment:
Can you update this and give name it, imageURL instead of photo, because photo can be anything but, like png,jpeg

Reply:
Try and upload the image to an image hosting site like amazon s3 or cloudinary and get the image url and save it to the db